src/.nuget/Microsoft.NETCore.ILAsm/runtime.FreeBSD.Microsoft.NETCore.ILAsm.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
(nit) `.props` files don't need to set `DefaultTargets`. It is just cruft that doesn't do anything.